viewing_party/party.py

```diff
@@ -1,23 +1,178 @@
+from collections import Counter
+
+
 # ------------- WAVE 1 --------------------
 
+
+# creates a movie 
+# returns dictionary
+
 def create_movie(title, genre, rating):
-    pass
+
+    if title == None or genre == None  or rating == None:
+        movie_dict = None
+    else:
+        movie_dict = {"title":title,"genre":genre,"rating":rating}
+
+    return movie_dict

```

Comment:
Nice work handling for returning "None" for missing data! In this case we are checking the `falsy` twice and in three different conditionals so we would want to use is rather than three `if` statements for comparison, we could simply check for the `falsy` value and return None in one line.

Something else we can consider: here we're explicitly checking if the values are None, but what if we got empty strings for one of the values? If we wanted to check for those at the same time as we check for None, we could rely on the truthy/falsy values of the parameters and write something like: `if not title or not genre or not rating:`